Repeat/repeat.py

```diff
@@ -0,0 +1,56 @@
+
+from PIL import Image
+import numpy as np     # numpy
+from collections import namedtuple
+
+repeat = namedtuple('repeat', 'count pixel')
+
+class Repeat:
+
+	def __init__(self,count1,pixel2):
+		self.r = repeat(count = count1, pixel = pixel2)
+
+#	def count(self):
+#		return self[0]
+#	def pixel(self):
+#		return self[1]
+
+	def generate(self):
+		"""  returns a list repeated pixel_list count number of times """		
+		mylist = []
+		pixel_list = []
+		count = self.r.count
+		pixel = self.r.pixel
+
+		if(type(pixel)==list):
+			if(type(pixel[0])==int): 
+				pixel_list = pixel 
+			else:
+				for j in range(len(pixel)):
+					pixel_list = pixel_list + pixel[j].generate()
+		else:
+			pixel_list = pixel.generate()	
+
+		for i in range(count):
+			for j in range(len(pixel_list)): 
+				mylist.append(pixel_list[j])	
+		return mylist	
+
+def to_matrix(list_1D,row_length):
```

Comment:
I don't think you need these functions - take a look at numpy's `reshape` function http://docs.scipy.org/doc/numpy-1.10.4/reference/generated/numpy.reshape.html